Power Up Proto/src/org/usfirst/frc/team5493/robot/commands/ClimbUp.java

```diff
@@ -1,6 +1,7 @@
 package org.usfirst.frc.team5493.robot.commands;
 
 import org.usfirst.frc.team5493.robot.Robot;
+import org.usfirst.frc.team5493.robot.subsystems.OneClimbyBoi;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.usfirst.frc.team5493.robot.subsystems.OneClimbyBoi'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)